src/module_wrap.cc

```diff
@@ -461,7 +461,6 @@ URL Resolve(std::string specifier, URL* base, bool read_pkg_json) {
   } else {
     return resolve_module(specifier, base);
   }
-  return URL("");
```

Comment:
Can you replace this with an `UNREACHABLE()`?

Reply:
added

Reply:
Hm, would it not even be better to just remove the else instead and have the regular return afterwards?